### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by pparent76** [fastcoding/mt7628ap#1]
Could you compile a mt7628 driver for openwrt 15.05 or lede 17.01?

**Comment by fastcoding** [comment]
yes, I did that for openwrt 15.05 in my spare time. but I got no time to test it yet. I will update this once  tested ok.

**Comment by pparent76** [comment]
Hi, thanks for the answer. We are looking for a reliable wifi driver for mt7628 on openwrt 15.05.

If you think you are able to do it we can pay you for it.

Please contact us at tech@comminter.com

**Comment by fastcoding** [comment]
Thanks for your suggestion. It is attractive but I'm still busy in my project. Fixing this may need to look into the details of how mt7628 wifi works. I will get back to this after I get enough spare time.
